.github/workflows/build-domain.yaml

```diff
@@ -126,30 +127,43 @@ jobs:
           overrides="${service}/chart-overrides.yaml"
           name=$(basename $(yq '.name' "$overrides"))
 
-          version=$(yq '.version' "$overrides")
-          version=$(sh ./.github/tooling/semver.sh "$version" "$PR")
+          tag=$(yq '.version' "$overrides")
+          tag=$(sh ./.github/tooling/semver.sh "$tag" "$PR")
 
-          echo "service=${service}" >> $GITHUB_OUTPUT
-          echo "project=${PROJECT}" >> $GITHUB_OUTPUT
-          echo "docker=${docker}" >> $GITHUB_OUTPUT
-          echo "domain=${domain}" >> $GITHUB_OUTPUT
-          echo "version=${version}" >> $GITHUB_OUTPUT
-          echo "name=${name}" >> $GITHUB_OUTPUT
+          dotnet=${{ matrix.build.dotnetVersion }}
+
+          {
+            echo "service=${service}"
+            echo "project=${PROJECT}"
+            echo "docker=${docker}"
+            echo "domain=${domain}"
+            echo "tag=${tag}"
+            echo "name=${name}"
+            echo "dotnet=${dotnet::${#dotnet}-2}"
```

Comment:
whats the -2 for? :)

Reply:
Almost, it turns `7.0.201` into `7.0.2`. The first one being the SDK version and the later one being the runtime version.